### PR TITLE
fix(workflows): Do not run workflows for direct pushes and merges

### PR DESCRIPTION
**Pull request by ChristophWurst**
Checking pull requests should be sufficient. Some workflows already did that. Now it is consistent.

This has the potential of freeing a lot of CI time.

### CODE REVIEW

File: workflow-templates/lint-info-xml.yml [hunk 1]
```diff
@@ -7,7 +7,6 @@ name: Lint
 
 on:
   pull_request:
-  push:
     branches:
```
Comment (ChristophWurst) [inline workflow-templates/lint-info-xml.yml:10]: I am aware that the branches moved up from push to pull_request. this just expresses that only PRs aganist main, master and stable* should be built, which is reasonable.

**Comment by nickvergessen** [comment]
I thought we decided to at least run one unit test (as installation test) on push, so the translation updates can give an indicator on errors?

**Comment by ChristophWurst** [comment]
I thought we concluded that nobody would check the results anyway? :see_no_evil:

**Comment by juliushaertl** [comment]
> I thought we decided to at least run one unit test (as installation test) on push, so the translation updates can give an indicator on errors?

Would agree that this is quite useful, especially when tests in apps start to fail due to server changes.

**Pull request closed without merging**